packages/video_player/example/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  video_player: ^2.4.7
+  video_player: ^2.7.2
```

Comment:
Have you checked the differences from v2.4.7 to v2.7.2 in video_player?

https://github.com/flutter/packages/blob/619af75f7966f90498dfce3a754d37422b972c6d/packages/video_player/video_player/CHANGELOG.md

Reply:
Okay I have gone to v2.6.0 (as that is the exact one that enables `isPlaying`, instead of jumping to v2.7.2. The `isComplete` from v2.7.2 works but there is additional work needed with deprecating `VideoPlayerController.network` so I can upgrade that in another pull request in the future.